apps/client/sttr_build/index.js

```diff
@@ -67,7 +67,7 @@ function checkStatus(res) {
       return json.map(({ _id }) => _id);
```

Comment:
Is there any advantage to also write the version in the `topics.source.json` file?